test/interop/C/errorMessage/initTwice.cleanfiles

```diff
@@ -0,0 +1,2 @@
+lib/libdefaultValueWarning.a
+lib/defaultValueWarning.h
```

Comment:
Note that since cleanfiles are run at the start of a directory rather than right before a test run, and this library is used in multiple tests, it may be created by an earlier test run.  However, I don't think that'll be a problem unless we try to compile it differently in the other test.  So maybe it would be worth adding a note to this precomp and the precomp for `use_defaultValueWarning` to alert anyone modifying either of them of the impact on the other (or maybe make the `use_defaultValueWarning` precomp a link to this one instead?)

Reply:
Good point.  Independently of this comment, I was feeling a little dirty using another test's file (and one that generated a warning) as it was, so I ended up just creating my own file.  In the original draft, I wasn't even sure you were on-board with the proposed approach, so was trying to put minimal time into writing tests (to the extent that I failed to even write a test checking that the finalize didn't exit...!)

Reply:
Sounds good :)